internal/resources/resource.go

```diff
@@ -23,6 +23,17 @@ const (
 	interconnectTLSVolumeName = "interconnect-tls-volume"
 	datastreamsTLSVolumeName  = "datastreams-tls-volume"
 
+	initMainSharedCertsVolumeName = "init-main-shared-certs-volume"
```

Comment:
I had gigantic trouble thinking about newly introduced names. I think there may be better (and shorter) options for some of them.